search/result.cpp

```diff
@@ -313,4 +317,15 @@ void AddressInfo::Clear()
   m_types.clear();
 }
 
+string DebugPrint(Result const & r)
```

Comment:
I would not like it. It's better to create a separate method like ToStringForStats() or something and do not touch DebugPrint().